File: CS/dxSample/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraReports.UI;

namespace dxSample {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
        }

        private void simpleButton1_Click(object sender, EventArgs e) {
            ShowReportPreview();
        }
        private static void ShowReportPreview() {
            new ReportPrintTool(new XtraReport1()).ShowPreview();
        }
    }
}

File: VB/dxSample/Form1.vb
Imports System
Imports DevExpress.XtraReports.UI

Namespace dxSample

    Public Partial Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs)
            Call ShowReportPreview()
        End Sub

        Private Shared Sub ShowReportPreview()
            Call New ReportPrintTool(New XtraReport1()).ShowPreview()
        End Sub
    End Class
End Namespace
